NEWS.md

```diff
@@ -1,251 +0,0 @@
-## DataFrames v0.6.11 Release Notes
```

Comment:
I could add it back in just to make the diff cleaner, but we haven't updated it in over a year and it should probably be removed anyways.